core/src/main/scala/org/apache/spark/memory/MemoryManager.scala

```diff
@@ -155,4 +164,217 @@ private[spark] abstract class MemoryManager extends Logging {
     _storageMemoryUsed
   }
 
+  // -- The code formerly known as ShuffleMemoryManager --------------------------------------------
+
+  /*
+   * Allocates a pool of memory to tasks for use in shuffle operations. Each disk-spilling
```

Comment:
Oh, right. I just wholesale copied this while merging the managers, so I still need to edit and lift these comments higher up.